build_library/build_image_util.sh

```diff
@@ -894,3 +882,66 @@ EOF
   cleanup_mounts "${root_fs_dir}"
   trap - EXIT
 }
+
+sbsign_image() {
+  local image_name="$1"
+  local disk_layout="$2"
+  local root_fs_dir="$3"
+  local image_kernel="$4"
+  local pcr_policy="$5"
+  local image_grub="$6"
+
+  local disk_img="${BUILD_DIR}/${image_name}"
+  local EFI_ARCH
+
+  case "${FLAGS_board}" in
+    amd64-usr) EFI_ARCH="x64" ;;
+    arm64-usr) EFI_ARCH="aa64" ;;
+  esac
```

Comment:
Maybe add `*) die "Unknown board ${BOARD@Q}" ;;` here, so @ader1990 will know about another place to modify for the riscv support. ;)